starlark/eval.go

```diff
@@ -435,13 +435,21 @@ func CompiledProgram(in io.Reader) (*Program, error) {
 // executes the toplevel code of the specified program,
 // and returns a new, unfrozen dictionary of the globals.
 func (prog *Program) Init(thread *Thread, predeclared StringDict) (StringDict, error) {
-	toplevel := makeToplevelFunction(prog.compiled, predeclared)
+	module, err := prog.InitModule(thread, predeclared)
```

Comment:
Let's revert these changes (LL438-452) as there's no need to publish InitModule.